pyuntl/dc_structure.py

```diff
@@ -60,38 +66,36 @@ def __init__(self, **kwargs):
             self.content = self.resolver(vocab_data, 'url')
 
     def add_child(self, child):
-        """This adds a child object to the current one.  It will check the
-        contained_children list to make sure that the object is allowable, and
-        throw an exception if not"""
-        # Make sure the child exists before adding it
+        """This adds a child object to the current one.  It will check
+        the contained_children list to make sure that the object is
+        allowable, and throw an exception if not."""
```

Comment:
Here a multi-line docstring has it's closing triple-quotes in-line, while in other places it has been placed on the next line. I believe that it should be on the next line in these cases.